pkg/gui/controllers/helpers/patch_building_helper.go

```diff
@@ -73,7 +73,7 @@ func (self *PatchBuildingHelper) RefreshPatchBuildingPanel(opts types.OnFocusOpt
 	ref := self.c.Contexts().CommitFiles.CommitFileTreeViewModel.GetRef()
 	to := ref.RefName()
 	from, reverse := self.c.Modes().Diffing.GetFromAndReverseArgsForDiff(ref.ParentRefName())
-	diff, err := self.c.Git().WorkingTree.ShowFileDiff(from, to, reverse, path, true, self.c.State().GetIgnoreWhitespaceInDiffView())
+	diff, err := self.c.Git().WorkingTree.ShowFileDiff(from, to, reverse, path, true, false)
```

Comment:
Worth adding a comment explaining why we don't allow ignoring whitespace here

Reply:
Added in 881a1c7cf61d.